india_compliance/gst_india/constants/__init__.py

```diff
@@ -36,6 +36,116 @@
     "WP",  # With Payment of Tax [1]
 )
 
+GST_DETAILS_FIELDS = [
```

Comment:
you can directly use already defined fields

Reply:
Yes these fields were used in many places, so defined it here and used it everywhere.